Fix the fetching of images in `CF_DIB` format in `DisplayServerWindows::clipboard_get_image`

FaycalElOuariachi opened the pull request:
A bug was occuring when trying to import an image from the windows clipbpoard, if it's in CF_DIB format (e. g. by taking screenshots). No image was returned.

It comes from a tiny error : in `DisplayServerWindows::clipboard_get_image`, `Image::create_from_data` was used with an instance of `Image`, but since it's a static function, it returns a new instance instead of modifying the 'called' one.
Correcting this fixes the importation.

### Testing with a GDScript

```
extends TextureRect

func _ready() -> void:
	if DisplayServer.clipboard_has_image() :
		var image = DisplayServer.clipboard_get_image()
		texture = ImageTexture.create_from_image(image)
```

This script was attached to a TextureRect node in the testing scene.
(At the start, the clipboard must contains an image in CF_DIB format, for instance a screenshot, or some images coming from a web navigator, in the right format, copied with the right-click context menu.)

PS : I apologies if my PR isn't conform to godot contributor's workflow, or for any other problem.

AThousandShips commented:
Your commit seems not to be linked to your GitHub account. See: [Why are my commits linked to the wrong user?](https://docs.github.com/en/pull-requests/committing-changes-to-your-project/troubleshooting-commits/why-are-my-commits-linked-to-the-wrong-user) for more info.

AThousandShips commented:
Seems good now, it's got two authors though, unsure if it matters, could do `git commit --amend --reset-author`

FaycalElOuariachi committed:
Fix the fetching of images in CF_DIB format in DisplayServerWindows::clipboard_get_image

Fix the fetching of images from windows clipboard, if they're in CF_DIB format (e. g. by taking screenshots).

Image::create_from_data was used with an instance of Image, but it's a static function, returning a new instance.

FaycalElOuariachi commented:
The amend was creating a new commit and a merge commit... Unfortunalty to avoid that, I did something bad by force pushing. Sorry for that... (I'll make sure to be cautious next time)

akien-mga commented:
It looks good now!

akien-mga commented:
Cherry-picked for 4.2.2.
